src/Commands/BuildToolsBase.php

```diff
@@ -815,7 +815,11 @@ public function pushCodeToPantheon(
         // checkout in a CI environment.)
         $this->passthru("git -C $repositoryDir checkout -B $branch");
         if ($prepareForPantheon) {
-          $this->passthru('composer prepare-for-pantheon');
+          $this->log()->notice('Determine whether prepare-for-pantheon exists for {site}', ['site' => $site]);
+          exec("composer help build-assets", $outputLines, $status);
```

Comment:
This should be `composer help prepare-for-pantheon`, not `composer help build-assets`

Reply:
Doh! thanks for catching that.